tests/arborist/test_urls.py

```diff
@@ -205,3 +206,68 @@ async def test_update_user_raises_error(
             )
         else:
             response = arborist_client.update_user(username, new_username=new_username)
+
+
+async def test_auth_request_positive(arborist_client, mock_arborist_request, use_async):
+    mock_arborist_request({"/auth/request": {"POST": (200, {"auth": True})}})
+    if use_async:
+        assert await arborist_client.auth_request(
+            "", "fence", "file_upload", "/data_upload"
+        )
+    else:
+        assert (
+            arborist_client.auth_request("", "fence", "file_upload", "/data_upload")
+            is True
+        )
+
+
+async def test_can_user_access_resources(
+    arborist_client, mock_arborist_request, use_async
+):
+    with pytest.raises(
+        AssertionError, match="'username' and 'jwt' cannot both be provided"
+    ):
+        await arborist_client.can_user_access_resources(
+            username="test-user",
+            jwt="abc",
+            resources={"/a": {"service": "service1", "method": "read"}},
+        )
+
+    mock_arborist_request(
+        {
+            "/auth/mapping": {
+                "POST": (
+                    200,
+                    {
+                        "/a": [{"service": "service1", "method": "read"}],
```

Comment:
Can we have any test cases involving the `*` scenarios for service or method?

Reply:
updated